src/de/jost_net/JVerein/gui/control/BeitragsgruppeControl.java

```diff
@@ -335,8 +461,32 @@ public TablePart getBeitragsgruppeTable() throws RemoteException
     }
     beitragsgruppeList.addColumn("Buchungsart", "buchungsart",
         new BuchungsartFormatter());
+    beitragsgruppeList.addColumn("Altersstaffel", "altersstaffel",
+        new JaNeinFormatter());
     beitragsgruppeList.addColumn("Notiz", "notiz", new NotizFormatter(40));
     beitragsgruppeList.setContextMenu(new BeitragsgruppeMenu());
+    beitragsgruppeList.setFormatter(new TableFormatter() {
+		
+		@Override
+		public void format(TableItem item) {
+			Beitragsgruppe b = (Beitragsgruppe)item.getData();
+			try {
+				if(b.getHasAltersstaffel())
+				{
+					String text = "";
+					DBIterator<Altersstaffel> it = b.getAltersstaffelIterator();
```

Comment:
Da ist mir wieder etwas passiert. Ich sollte nicht soviel herumspielen. Ich hatte ja einmal probiert die Altersstufen um eine Stufe zu verkleinern. Jetzt habe ich festgestellt, dass bei der Anzeige hier alle Stufen aus der DB angezeigt werden. Im Dialog habe ich aber eine weniger. Man dürfte wohl hier nur soviele anzeigen wie Stufen in den Einstellungen existieren.

Reply:
PS: Da sind sehr große Einrückungen im dem Codeabschnitt.

Reply:
Du testest aber auch wirklich alles! Habe ich behoben
Außerdem habe ich noch einen Test hinzugefügt damit sich Altersstufen nicht überschneiden.

PS. Bei der Formatierung fliegt bei mir immer mal wieder der Formatter für das Jverein Format raus, die tabs werden bei mir kleiner angezeigt und dadurch sieht es nicht nach so großen Einrückungen aus.